canopen/sdo/client.py

```diff
@@ -42,6 +45,7 @@ def __init__(self, rx_cobid, tx_cobid, od):
         self.responses = queue.Queue()
 
     def on_response(self, can_id, data, timestamp):
+        logger.debug(f"received response in {can_id} data {data}.")
```

Comment:
As I understand it, it is preferable to use old-style string interpolation for logging because then the final message will only be constructed if needed.

https://docs.python.org/3/howto/logging.html#logging-variable-data